studio/src/ui/Button.js

```diff
@@ -57,7 +57,7 @@ export class Button {
 
 		this.onClick = onClick;
 		this.boundClick = this.click.bind(this);
-		this.el.addEventListener("click", this.boundClick);
+		this.el.addEventListener("click", this.boundClick, true);
```

Comment:
The idea is to trigger the event callback on the capture phase before the event bubbles up through the DOM tree. 

I think it was actually a remnant of me trying to see if I could remove the `waitForEventLoop` promise in the `addPopover`/`createContextMenu` functions. I can remove this and the stopPropagation line since there isn't any difference from A/B testing.